### PR TITLE
 ZCS-14120: Translations for Hide in GAL

### PR DESCRIPTION
**Pull request by sumitTelus**
- Translation added for **Hide user in GAL** and **Hide aliases in GAL** label text.

### CODE REVIEW

File: WebRoot/messages/ZaMsg_ca.properties [hunk 1]
```diff
@@ -1187,7 +1187,8 @@ NAD_zimletStatus = Habilitat:
 
 NAD_CanonicalFrom = Adre\u00e7a can\u00f2nica:
 CannonicalWarning = Canviar l\u2019adre\u00e7a can\u00f2nica pot malmetre la ruta del correu i les cites de l\u2019agenda.
-LBL_zimbraHideInGal = Amaga a la llibreta global d\u2019adreces:
+LBL_zimbraHideInGal = Amaga l\u2019usuari a GAL:
+LBL_zimbraHideAliasesInGal = Amaga els àlies a GAL
```
Comment (sakshirawat0247) [inline WebRoot/messages/ZaMsg_ca.properties:1191]: à needs to be converted

File: WebRoot/messages/ZaMsg_pl.properties [hunk 2]
```diff
@@ -1181,7 +1181,8 @@ NAD_zimletStatus = W\u0142\u0105czone:
 
 NAD_CanonicalFrom = Adres kanoniczny:
 CannonicalWarning = Modyfikacja adresu kanonicznego mo\u017ce zak\u0142\u00f3ci\u0107 obieg wiadomo\u015bci i wydarze\u0144 w kalendarzu!
-LBL_zimbraHideInGal = Ukryj w GLA: 
+LBL_zimbraHideInGal = Ukryj użytkownika w GAL:
```
Comment (sakshirawat0247) [inline WebRoot/messages/ZaMsg_pl.properties:1184]: ż.      to be converted

File: WebRoot/messages/ZaMsg_ca.properties (hunk 1 above)
Comment (sumitTelus) [reply WebRoot/messages/ZaMsg_ca.properties]: Done

File: WebRoot/messages/ZaMsg_pl.properties (hunk 2 above)
Comment (sumitTelus) [reply WebRoot/messages/ZaMsg_pl.properties]: Done

**Review by silentsakky — APPROVED** [review]

File: WebRoot/messages/ZaMsg.properties [hunk 3]
```diff
@@ -1197,7 +1197,8 @@ NAD_zimletStatus = Enabled:
 
 NAD_CanonicalFrom = Canonical address:
 CannonicalWarning = Modifying canonical address may break mail routing and calendar appointments!
-LBL_zimbraHideInGal = Hide in GAL:
+LBL_zimbraHideInGal = Hide user in GAL:
+LBL_zimbraHideAliasesInGal = Hide aliases in GAL
```
Comment (k-kato) [inline WebRoot/messages/ZaMsg.properties:1201]: Just to confirm. A colon `:` is unnecessary in the value of `LBL_zimbraHideAliasesInGal`, correct?

Comment (sumitTelus) [reply WebRoot/messages/ZaMsg.properties:1201]: > Just to confirm. A colon `:` is unnecessary in the value of `LBL_zimbraHideAliasesInGal`, correct?

According to the ZCS-13220 mockup, the colon was not mentioned.

Comment (k-kato) [reply WebRoot/messages/ZaMsg.properties:1201]: okay

**Review by k-kato — APPROVED** [review]

**Review by sakshirawat0247 — APPROVED** [review]

**Pull request merged**